.github/workflows/integration-tests.yml

```
@@ -0,0 +1,66 @@
+name: Integration Tests
+
+on:
+  pull_request:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 4 * * *"
+  push:
+    tags:
+      - "*"
+    branches:
+      - main
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  ALLOW_PLOTTING: true
+  SHELLOPTS: 'errexit:pipefail'
+
+jobs:
+  # For now this is just MNE-Python, but others could be added
+  mne:
+    name: MNE-Python
+    runs-on: ubuntu-22.04
+    env:
+      DISPLAY: ':99.0'
+      MNE_LOGGING_LEVEL: 'info'
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
+      - run: git clone --depth=1 https://github.com/mne-tools/mne-python.git --branch main --single-branch
+      - run: ./mne-python/tools/setup_xvfb.sh
+      - name: Install PyVista
+        run: pip install -ve .  # pyvista
+      - name: Install MNE dependencies
+        run: ./tools/github_actions_dependencies.sh
+        working-directory: mne-python
+      - name: Install MNE
+        run: pip install -ve .  # mne
+        working-directory: mne-python
+      - name: MNE Info
+        run: mne sys_info
```

If you just do a `-p` here, it will show the installation dir for `pyvista`, no need fro the `pip show pyvista` below
```suggestion
        run: mne sys_info -p
```